bindata/network/kuryr/005-daemon.yaml

```diff
@@ -0,0 +1,173 @@
+          oc config "--config=${file}" view --flatten > /tmp/kubeconfig
+          oc config --config=/tmp/kubeconfig set-credentials sa "--token=$( cat /var/run/secrets/kubernetes.io/serviceaccount/token )"
+          oc config --config=/tmp/kubeconfig set-context "$( oc config --config=/tmp/kubeconfig current-context )" --user=sa
+
+          # Launch the SkyDNS
+          exec openshift start network --enable=dns --config=/etc/origin/node/node-config.yaml --kubeconfig=/tmp/kubeconfig --loglevel=${DEBUG_LOGLEVEL:-2}
+
+        securityContext:
+          privileged: true
+          runAsUser: 0
+        env:
+        - name: OPENSHIFT_DNS_DOMAIN
+          value: cluster.local
+        volumeMounts:
+        - name: host-config
+          mountPath: /etc/origin/node
+          readOnly: true
+        - mountPath: /etc/sysconfing/origin-node
+          name: host-sysconfig-node
+          readOnly: true
+        - mountPath: /var/run
+          name: host-var-run
+      - name: kuryr-cni
+        image: {{ .DaemonImage }}
+        command: [ "cni_ds_init" ]
+        env:
+        - name: CNI_DAEMON
+          value: "True"
+        - name: KUBERNETES_NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        - name: KURYR_CNI_POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - name: bin
+          mountPath: /opt/cni/bin
+        - name: net-conf
+          mountPath: /etc/cni/net.d
+        - name: config-volume
+          mountPath: /etc/kuryr/kuryr.conf
+          subPath: kuryr.conf
+        - name: proc
+          mountPath: /host_proc
+        - name: openvswitch
+          mountPath: /var/run/openvswitch
+{{ if (default .DaemonEnableProbes false) eq "true" }}
+        readinessProbe:
+          httpGet:
+            path: /ready
+            port: {{ default .DaemonProbesPort 8090 }}
+            scheme: HTTP
+          initialDelaySeconds: 15
+          timeoutSeconds: 5
+        livenessProbe:
+          httpGet:
+            path: /alive
+            port: {{ default .DaemonProbesPort 8090 }}
+          initialDelaySeconds: 15
+{{ end }}
+      volumes:
+        - name: bin
+          hostPath:
+            path: {{ default .CNIBinDir "/opt/cni/bin" }}
+        - name: net-conf
+          hostPath:
+            path: /etc/cni/net.d
+        - name: config-volume
+          configMap:
+            name: kuryr-config
+        - name: proc
+          hostPath:
+            path: /proc
+        - name: openvswitch
+          hostPath:
+            path: /var/run/openvswitch
+        - name: host-config
+          hostPath:
+            path: /etc/origin/node
+        - name: host-sysconfig-node
+          hostPath:
+            path: /etc/sysconfig/origin-node
+        - name: host-var-run
```

Comment:
If you can get rid of bind-mounting `/var/run`, that would be good. Someday I'll get rid of it for openshift-sdn...

Reply:
@tomassedovic Hey, do you remember why this /var/run mount is required for SkyDNS?